go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/gorilla/websocket v1.4.1
 	github.com/iancoleman/strcase v0.0.0-20191112232945-16388991a334
 	github.com/kr/text v0.1.0
-	github.com/mholt/archiver v2.1.0+incompatible
+	github.com/mholt/archiver v3.1.1+incompatible
```

Comment:
We froze archiver on v2.1.0 due to https://github.com/mholt/archiver/issues/152. However, it should be fixed in https://github.com/mholt/archiver/issues/153, although it looks like I never got around to testing the fix (whoops). So we should certainly test that.

Reply:
> We froze archiver on v2.1.0 due to [mholt/archiver#152](https://github.com/mholt/archiver/issues/152). However, it should be fixed in [mholt/archiver#153](https://github.com/mholt/archiver/pull/153), although it looks like I never got around to testing the fix (whoops). So we should certainly test that.

How ?

Reply:
I tested. It doesn't work. I created https://github.com/mholt/archiver/issues/213.

Reply:
So basically no support for tar.xz then.

Reply:
I see some code for `tar.xz` in version 2.1 [here](https://github.com/mholt/archiver/blob/v2.1/tarxz.go#L12-L14) but I haven't investigated if de-archiving is supported, or how to do it.